assets/src/design-system/components/contextMenu/contextMenu.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+/**
+ * Internal dependencies
+ */
+import { Popover, Shadow } from './styled';
+import Menu, { MenuPropTypes } from './menu';
+
+const ContextMenu = ({ isOpen, items, ...props }) => {
+  return (
+    <Popover role="dialog" isOpen={isOpen} {...props}>
+      <Menu items={items} />
+      <Shadow />
```

Comment:
Added the `Shadow` component from the old context menu. This is for the animation since the shadow used to be animated.